src/Data/ProgrammesDb/EntityRepository/ServiceRepository.php

```diff
@@ -53,4 +54,16 @@ public function findAllInNetworks(): array
 
         return $qb->getQuery()->getResult(Query::HYDRATE_ARRAY);
     }
+
+    public function findAllInNetwork(Nid $networkId): array
+    {
+        $qb = $this->createQueryBuilder('service')
+            ->select('service', 'network')
```

Comment:
createQueryBuilder automatically adds the service to the select clause, so you don't need to override the whole thing here. Instead do 

```php
$this->createQueryBuilder('service')
    ->addSelect('network')
```